strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/BaseController.java

```diff
@@ -297,8 +299,20 @@ protected void copyToResponse(InputStream is,
             while ((readLength = is.read(bytes, 0, bytes.length)) != -1)
             {
                 // Write the artifact
-                os.write(bytes, 0, readLength);
-                os.flush();
+                try
+                {
+                    os.write(bytes, 0, readLength);
+                    os.flush();
+                }
+                catch (EOFException e)
+                {                   
+                    if (e.getCause().getMessage().equals("Connection reset by peer")) {
+                        logger.debug("Socket has been closed. Possibly, user cancelled download.");
+                        // Set response status 202 ACCEPTED just for logs
+                        response.setStatus(202);
+                        break;
+                    }
```

Comment:
otherwise we should re-throw the `EOFException` rather than do nothing

Reply:
Thanks! I'll fix this.

Reply:
Done

Reply:
`else` is unnecessary